### ISSUE TITLE
i3-input ignores input when launched from i3

### ISSUE THREAD

**Issue by i3bot** [i3/i3#175]
[Originally reported by anonymous]
(1. Add binding to i3's configuration to exec i3-input

```
  bindsym Mod1+z exec i3-input
```
1. Execute the keybinding
2. i3-input doesn't have the keyboard focus and doesn't accept input

Expected:

i3-input should have the keyboard focus

A workaround is to use a wrapper script to sleep before executing i3-input.
